tests/unit/test-cmd/test.build.js

```diff
@@ -28,32 +27,25 @@ const log = createLogger(__filename);
 
 describe('build', () => {
 
-  it('zips a package', () => {
-    const zipFile = new ZipFile();
-
-    return withTempDir(
-      (tmpDir) =>
-        build({
-          sourceDir: fixturePath('minimal-web-ext'),
-          artifactsDir: tmpDir.path(),
-        })
-          .then((buildResult) => {
-            assert.match(buildResult.extensionPath,
-                         /minimal_extension-1\.0\.zip$/);
-            return buildResult.extensionPath;
-          })
-          .then((extensionPath) => zipFile.open(extensionPath))
-          .then(() => zipFile.extractFilenames())
-          .then((fileNames) => {
-            fileNames.sort();
-            assert.deepEqual(fileNames,
-                             ['background-script.js', 'manifest.json']);
-            return zipFile.close();
-          })
-    );
-  });
+  it('zips a package', () => withTempDir(
+    async (tmpDir) => {
+      const zipFile = new ZipFile();
+      const buildResult = await build({
+        sourceDir: fixturePath('minimal-web-ext'),
+        artifactsDir: tmpDir.path(),
+      });
+      assert.match(buildResult.extensionPath,
+                   /minimal_extension-1\.0\.zip$/);
+      const extensionPath = await buildResult.extensionPath;
```

Comment:
@saintsebastian no need to `await` here, `buildResult.extensionPath` should already be a string and you can pass it directly to zipFile.open at line 40.